src/cli.rs

```diff
@@ -126,6 +127,14 @@ fn resynth() -> Result<(), ()> {
                 .action(ArgAction::SetTrue)
                 .help("Keep pcap files on error"),
         )
+        .arg(
+            Arg::new("docs")
+                .long("output-docs")
+                .value_name("FILE")
+                .required(false)
+                .value_parser(value_parser!(PathBuf))
+                .help("Output documentation"),
+        )
```

Comment:
Should this arg be mutually exclusive with the `in` arg? It looks like if this is set we short-circuit but we still require an input file

Reply:
Something like the following, chained onto the end could work:
```rust
        .group(
            ArgGroup::new("run_mode")
                .args(["docs", "in"])
                .required(true),
        );
```

although it will nullify the `required` fields of both these args